content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -114,3 +114,61 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`token`]: https://hl7.org/fhir/R4/search.html#token
 [errors]: ../../#client-errors
 [OperationOutcomes]: https://hl7.org/fhir/R4/operationoutcome.html
+
+## Create
+
+Create new allergy.
+
+    POST /AllergyIntolerance
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:allergy_intolerance, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance
+
+#### Body
+
+<%= json(:r4_allergy_intolerance_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Wed, 14 Aug 2019 17:23:14 GMT
+Etag: W/"12793861"
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/6767735,
```

Comment:
Since i created this before sandbox deployment, i believe i just copied things over. 
Updated Etag: https://github.com/cerner/fhir.cerner.com/pull/324/commits/af5ece927b9b9ffd2a3f489c1b4bf0b8fc389518#diff-7ae7d3678162c287b6daeaf7f84f7e06R158